### ISSUE TITLE
Add focus ability and style to Contact Us links for accessibility

### ISSUE THREAD

**Issue by arasantos** [umwics/WICS-website-vue#77]
tabIndex allows users to navigate the interactive elements in Contact Us page using tab. 

Focus style refers to making it visible to the user that they have focused or "tabbed" on a link option. Usually, this keeps the active style PLUS a blue outline around the link being focused on. There is also usually an outline offset.

The focus style will likely have similarities to the navbar's focus style, so it can be a global style in App.vue if it doesn't exist already. Name it `link-focus`

Maybe look into making a reusable component for the contact us links since they are used 4 times in the contact us page, and it's possible to use it in more in the future. This should be in a separate issue though.
